stdlib/LinearAlgebra/src/qr.jl

```diff
@@ -542,6 +542,36 @@ function rank(A::QRPivoted; atol::Real=0, rtol::Real=min(size(A)...) * eps(real(
     return something(findfirst(i -> abs(A.R[i,i]) <= tol, 1:m), m+1) - 1
 end
 
+function nullspace(A::QRPivoted; atol::Real=0, rtol::Real=min(size(A)...) * eps(real(float(one(eltype(A.Q))))) * iszero(atol))
+    m, n = size(A, 1), size(A, 2)
+    (m == 0 || n == 0) && return Matrix{eigtype(eltype(A.Q))}(I, n, n)
+
+    indstart = rank(A; atol, rtol) + 1
+    return A.Q[:, indstart:end]
+end
+
+function cond(A::QRPivoted, p::Real=2)
+    m = min(size(A.R)...)
+    if p == 2
+        maxv = abs(A.R[1,1])
+        return iszero(maxv) ? oftype(real(maxv), Inf) : maxv / abs(A.R[m,m])
+    elseif p == 1 || p == Inf
+        checksquare(A.R)
+        R = UpperTriangular(A.R)
+        try
+            Rinv = inv(R)
+            return opnorm(R, p)*opnorm(Rinv, p)
```

Comment:
This algorithm also seems wrong?

I'm not there that there is any good way to compute the p=1 or Inf condition numbers from the QR factorization?  Might better to just throw an error in this case, since the user is better off calling `cond` on the original matrix AFAICT.